pages/Livesite/announcements.js

```diff
@@ -1,24 +1,178 @@
+      'emphasis',
+      'link',
+      'break',
+      'list',
+      'listItem',
+    ]}
   >
-    <img
-      src="https://i.imgur.com/lEjmB7U.gif"
-      width="480"
-      height="362"
-      alt="panik"
-    />
-    <p>Announcements - This area under construction.</p>
-  </Page>
+    {content}
+  </ReactMarkdown>
 );
 
+const announcementDateFormat = (timestamp) => {
+  const options = {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+    hour: '2-digit',
+    minute: '2-digit',
+  };
+  return new Date(timestamp).toLocaleDateString('en-US', options);
+};
+
+export default ({ hackathons }) => {
+  const [announcements, setAnnouncements] = useState([]);
+  const [activeHackathon, setActiveHackathon] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+  const [activeModal, setActiveModal] = useState('');
+  const [currentAnnouncement, setCurrentAnnouncement] = useState({});
+  const { email: user } = useAuth().user;
+
+  const getAsyncData = async () => {
+    setActiveHackathon(await getActiveHackathon);
+    setIsLoading(false);
+  };
+
+  useEffect(() => {
+    getAsyncData();
+  });
+
+  // eslint-disable-next-line consistent-return
+  useEffect(() => {
+    if (activeHackathon) {
+      return subscribeToLivesiteAnnouncements(activeHackathon, (data) => {
+        setAnnouncements(data);
+      });
+    }
+  }, [activeHackathon, setAnnouncements]);
+
+  const handleNew = () => {
+    setCurrentAnnouncement({ editor: user });
+    setActiveModal(NEW);
+  };
+
+  const handleDelete = (key) => {
+    // eslint-disable-next-line no-restricted-globals
+    if (confirm('Are you sure? This cannot be undone')) {
+      deleteAnnouncement(activeHackathon, key);
+    }
+  };
+
+  const handleEdit = (key) => {
+    setCurrentAnnouncement({ ...announcements[key], editor: user, id: key });
+    setActiveModal(EDIT);
+  };
+
+  const handleCloseModal = () => {
+    if (currentAnnouncement.content) {
+      // eslint-disable-next-line no-restricted-globals
+      if (confirm('Are you sure? You will lose all progress')) {
+        setActiveModal('');
+      }
+    } else {
+      setActiveModal('');
+    }
+  };
+
+  const handleSave = () => {
+    updateAnnouncement(activeHackathon, currentAnnouncement);
+    setActiveModal('');
+  };
+
+  const changeCurrentAnnouncement = (content) => {
+    setCurrentAnnouncement({ ...currentAnnouncement, content });
+  };
+
+  return (
+    <Page
+      currentPath="Livesite"
+      hackathons={hackathons}
+      navbarItems={LIVESITE_NAVBAR}
+    >
+      <AnnouncementsCard
+        isLoading={isLoading}
+        announcements={announcements}
+        handleNew={handleNew}
+        handleDelete={handleDelete}
+        handleEdit={handleEdit}
+      />
+
+      <Modal
+        isOpen={activeModal === EDIT}
+        handleClose={handleCloseModal}
+        handleSave={handleSave}
+        modalAction={EDIT}
+      >
+        <div>
+          <strong>Announcement Content</strong>
```

Comment:
nit: `<h3>` might be better here?
https://webmasters.stackexchange.com/questions/111465/h3-vs-strong-for-faqs

Reply:
Following the other labels in cms and using strong